src/Presentation/Presenters/CancelDonationHtmlPresenter.php

```diff
@@ -22,7 +22,8 @@ public function __construct( TwigTemplate $template ) {
 	public function present( CancelDonationResponse $response ): string {
 		return $this->template->render( [
 			'donationId' => $response->getDonationId(),
-			'cancellationSuccessful' => $response->cancellationSucceeded()
+			'cancellationSuccessful' => $response->cancellationSucceeded(),
+			'mailDeliveryFailed' => !$response->mailDeliveryFailed()
```

Comment:
Dang, it was `mailDeliverySuccessful` first, then I last-minute-equalized, but forgot to remove the negation. Good, that this line of code is not tested. Fixed that, added a route test.